build.gradle

```diff
@@ -23,6 +24,7 @@ buildscript {
 
         // Automatic releases to Sonatype.
         classpath 'io.codearte.gradle.nexus:gradle-nexus-staging-plugin:0.12.0'     // https://github.com/Codearte/gradle-nexus-staging-plugin/releases
+        classpath "de.marcphilipp.gradle:nexus-publish-plugin:0.1.0"                // https://github.com/marcphilipp/nexus-publish-plugin/releases
```

Comment:
jeez, so it integrations with gradle-nexus-staging-plugin if it's present

ooof

Reply:
:wrench::hankey::hammer: